examples/jobs/passjob_html.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+import sys
```

Comment:
I didn't get why we are adding those example files. I mean... I understand that we are providing a use case example and it is handy, but shouldn't this be a unit test or a functional test?

Reply:
Maybe we can eliminate most of them once we document the configuration entries.  But maybe, we can also turn them into something testable.  For now, they are example files that can help users (myself included) to write jobs that do various things, for what that's worth.